app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -136,6 +137,22 @@ open class BpkSnapshotTest {
     }
   }
 
+  private fun View.findComposeView(): ComposeView? {
```

Comment:
this essentially loops through the view tree until we find the right view. we only have 1 compose view as a root for our tests